tsconfig.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "@linzjs/style/tsconfig.base.json",
   "compilerOptions": {
-    "lib": ["ES2022"],
-    "outDir": "build"
+    "lib": ["ES2022"]
```

Comment:
If the tsc is run this will now dump a bunch of `.js` `.js.map` and `.d.ts` into the folders,  I would prefer leaving this, or alternatively using `noEmit: true`